vms/platformvm/blocks/executor/acceptor.go

```diff
@@ -262,10 +214,19 @@ func (a *acceptor) proposalBlock(b blocks.Block) {
 	//   (The VM's Shutdown method commits the database.)
 	//   The snowman.Engine requires that the last committed block is a decision block
 
-	a.backend.lastAccepted = b.ID()
+	blkID := b.ID()
+	a.backend.lastAccepted = blkID
+
+	a.ctx.Log.Trace(
+		"accepted block",
+		zap.String("blockType", blockType),
+		zap.Stringer("blkID", blkID),
+		zap.Uint64("height", b.Height()),
+		zap.Stringer("parentID", b.Parent()),
+	)
```

Comment:
Because we don't actually write to state here it felt weird to log the `utxoChecksum`... but if people felt it should be there for consistency then it is easy to add.

Reply:
I think it would be nice to get the checksum from logs on each accepted block.